bundles/net.tourbook/src/de/byteholder/geoclipse/messages.properties

```diff
@@ -1,356 +1,356 @@
-TileInfo_Control_Pattern                     = Q:000  T:000.000.000  P:000.000.000  S:00.00 xx
-TileInfo_Control_Statistics_DownloadData     = Q:{0} - F:{1} - {2} kB
-TileInfo_Control_Statistics_DownloadDataFile = {1} - is downloading
-TileInfo_Control_Statistics_DownloadDataInit = Q:{0} - F:{1} - connecting {2}
-TileInfo_Control_Tooltip_Line1               = Q: Queued tiles\n\
-                                               
-TileInfo_Control_Tooltip_Line2               = T: Loading tiles:\u0009Loaded . Started . Errors\n\
-                                               
-TileInfo_Control_Tooltip_Line3               = P: Painting tiles:\u0009Finished . Started . Errors\n\
-                                               
-TileInfo_Control_Tooltip_Line4               = S: SRTM data:\u0009Loaded . Errors\n\
-                                               
-TileInfo_Control_Tooltip_Line5               = F: File download 
-TileInfo_Control_Tooltip_StatisticsAll       = Q:{0} - T:{1}.{2}.{3} - P:{4}.{5}.{6} - S:{7}.{8}
-TileInfo_Control_Tooltip_Title               = Map Tile Info\n\
-                                               \n\
-                                               
-
-TileInfo_Error_NoMapProvider = There is currently no map provider which provides a map tile image for this location and zoom level, the reason could be:\n\
-                               \n\
-                               - a map provider is not set to be displayed\n\
-                               \n\
-                               - all displayed map providers do not support a tile image for this location and zoom level
-
-TileInfo_Position_Latitude  = Lat: 
-TileInfo_Position_Longitude = Lon: 
-TileInfo_Position_X         = \ x:
-TileInfo_Position_Y         = \ y:
-TileInfo_Position_Zoom      = zoom:
-
-Url_Parameter_Random      = Random Number
-Url_Parameter_Random_Abbr = random
-Url_Parameter_Text        = Text
-Url_Parameter_Text_Abbr   = text
-Url_Parameter_X           = X Position
-Url_Parameter_X_Abbr      = x
-Url_Parameter_Y           = Y Position
-Url_Parameter_Y_Abbr      = y
-Url_Parameter_Zoom        = Zoom level
-Url_Parameter_Zoom_Abbr   = zoom
-
-Wms_ServerError_Info = \n\
-                       !!! Tip !!!\n\
-                       \n\
-                       Try to load the requested url in a web browser. Sometimes even the server exception xml contains errors which cannot be displayed here.
-
-geoclipse_extensions_loading        = Loading ...
-geoclipse_extensions_loading_failed = Loading Failed!
-
-tileInfo_action_clearStatistics = &Reset Statistics
+#Generated by Eclipse Messages Editor (Eclipse Babel)
+
+App_Dlg_ConfirmFileOverwrite_Message = Overwrite existing file \n\
+                                       \n\
+                                       {0} ? 
+App_Dlg_ConfirmFileOverwrite_Title   = File exists
+
+App_Error_TooManyWords = There are too many words to create a unique id

```

Comment:
> This file was not modified except the line delimiter was changed to the line delimiter CRLF. Why?
> 
> ![mt-line-delimiter](https://user-images.githubusercontent.com/1283445/108616976-ccd9ef00-7412-11eb-99e4-50e95496569a.png)
> 
> When I discover, that a file do not have the Unix line delimiter then I change it but this file was reverted back.
> 
> ![mt-line-delimiter-conversion](https://user-images.githubusercontent.com/1283445/108616977-cd728580-7412-11eb-96cc-9c80a72aecd1.png)

@wolfgang-ch 
that's very strange, I actually never opened that file. Is it possible that is coming from the PR you and Frederic did lately ? because I actually didn't touch this file.

Reply:
>@wolfgang-ch
>that's very strange, I actually never opened that file. Is it possible that is coming from the PR you and Frederic did lately ? because I actually didn't touch this file.

It's problematic to say, it's not from me when the git history shows the opposite, I found this history in your repo.

![mt-messages](https://user-images.githubusercontent.com/1283445/108673339-c6677800-74e3-11eb-8711-05a67c5a0036.png)